README.md

```diff
@@ -68,7 +68,6 @@ Current maintainers of this lesson (in alphabetical order) are:
 * [Steve Crouch][steve-crouch]
 * [Kamilla Kopec-Harding][kamilla-kopec-harding]
 * [Doug Lowe][doug-lowe]
-* [Sam Mangham][sam-mangham]
```

Comment:
Sorry to see you go @smangham . Hope all is well.

@anenadic shall we remove the reference link at the bottom of the page as well?